data-es6.js

```diff
@@ -6324,9 +6324,14 @@ exports.tests = [
       res: {
         babel6corejs2: babel.corejs,
         typescript1corejs2: typescript.corejs,
+        ie11: {
+          val: true,
+          note_id: "ie-clampedarray",
+          note_html: "A partial implementation of <code>Uint8ClampedArray</code> has been introduced with <a href='https://support.microsoft.com/en-us/help/2929437/description-of-the-security-update-for-internet-explorer-11-on-windows'>KB2929437</a>, so this test fails on early IE11 builds."
```

Comment:
Can you elaborate what does "partial implementation" mean? It would be great if we can add failing tests for those cases.

Reply:
To be honest, this was so long ago that I don´t remember what I meant.  But I believe that a better description wold be "A buggy implementation shipped with IE10 and early releases of IE11, and it was fixed with the PR".  Is this enough?
I don't know enough to write a new subtest, but I believe this one already fails on those cases...

Reply:
```suggestion
          note_html: "A buggy implementation of <code>Uint8ClampedArray</code> has been fixed with patch <a href='https://support.microsoft.com/en-us/help/2929437/description-of-the-security-update-for-internet-explorer-11-on-windows'>KB2929437</a>"
```

Reply:
Done.